app/SearchApi/BcGov.Fams3.SearchApi.Contracts/Person/ResponsePerson.cs

```diff
@@ -0,0 +1,34 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Text;
+
+namespace BcGov.Fams3.SearchApi.Contracts.Person
+{
+    public class ResponsePerson : PersonalInfo
```

Comment:
This response person seems like person contract ? Can we use person for this  and just extend to have missing fields ?

Reply:
the ResponsePerson here is only for some additional Person data in Response. It could be empty or a collection.